app/controllers/groups/invites_controller.rb

```diff
@@ -25,17 +25,24 @@ def create
     groups = @group.network? ? recipients.groups : []
     emails = recipients.emails
 
-    reqs = []; mailers = []
+    reqs, mailers = [], []
     unless users.any? or emails.any? or groups.any?
       raise_error('Recipient required')
     end
+
     users.each do |user|
       reqs << RequestToJoinUs.create(created_by: current_user,
         recipient: user, requestable: @group)
     end
+
     groups.each do |group|
-      reqs << RequestToJoinOurNetwork.create(created_by: current_user,
-        recipient: group, requestable: @group)
+      if council = group.council
+        reqs << RequestToJoinOurNetwork.create(created_by: current_user,
+          recipient: council, requestable: @group)
```

Comment:
> I confirmed that all group requests can only be approved by group admins. So only notifying them makes sense to me.

In this situation we send a notice to each group member, but only members of the council would be able to approve it, right?

That maybe it is better to update the `RequestNotice` model to handle it in such way:

```ruby
...
        elsif recipient.is_a? Group
          if recipient.council
            recipient.council.users.each do |user|
              create!(request: request, user: user)
            end
          else
            recipient.users.each do |user|
              create!(request: request, user: user)
            end
          end
...
```
where `request` - it is still `RequestToJoinOurNetwork` or any other, but notification would be created only to right users?
Because what is the reason to notify users that are not able to approve the request?